### PR TITLE
Dev yetfl

### PR DESCRIPTION
**Pull request by oftadehomid**
Hello Pierre!

And please do not forget to give a reference in ETFL's Readme to yetfl repository :)

Thank you,

### CODE REVIEW

**Comment by psalvy** [comment]
Thanks for the PR
Can you please restore the etfl/data folder so that the tests can be run by Travis
(see error https://travis-ci.com/github/EPFL-LCSB/etfl/builds/225189379)
Cheers,
Pierre

**Comment by oftadehomid** [comment]
I did it, but the Travis test again failed with an error about the version of xlrd which I do not have any idea about it.
Also, I restored two other test files that you had added more recently (test_thermo and test_resources_alloc)
Cheers,
Omid

File: README.rst [hunk 1]
```diff
@@ -1,73 +1,14 @@
 ETFL
 ====
-|Documentation Status| |Build Status| |Codecov| |Codacy branch grade| |license| |doi|
```
Comment (psalvy) [inline README.rst]: @oftadehomid your commit removes important documentation from the readme. Would you please fix this ? 
Could you also check that it does not break the documentation elsewhere?
Cheers,
Pierre

File: setup.py [hunk 2]
```diff
@@ -23,22 +23,23 @@
 
 setup(name='ETFL',
       version=version_tag,
-      author='Pierre SALVY',
+      author='Pierre SALVY, Omid OFTADEH',
       author_email='softwares.lcsb@epfl.ch',
       url='https://github.com/EPFL-LCSB/etfl/',
       download_url='https://github.com/EPFL-LCSB/etfl/archive/'+version_tag+'.tar.gz',
-      install_requires=['biopython',
+      install_requires=['biopython<=1.77',
```
Comment (psalvy) [inline setup.py]: Why this ?

**Pull request closed without merging**